templates/gallery/partials/header.snip.html

```diff
@@ -0,0 +1,49 @@
+{{!
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+{{#header}}
+  <header
+    class="
+      gallery-header
+      {{#invert}}gallery-base-invert{{/invert}}
+      {{#mobile-header}}gallery-mobile-header{{/mobile-header}}
+    "
+  >
```

Comment:
why is this formatted weird - the closing arrow is in a new line? don't line break on HTML it will render weird in the view source and it could also cause issues that are hard to debug.